daily_read/ngi_data.py

```diff
@@ -0,0 +1,81 @@
+import json
+import logging
+import os
+
+log = logging.getLogger(__name__)
+
+
+class ProjectDataMaster(object):
+    def __init__(self, config, sources):
+        self.config = config
+        self.sources = sources
+        self.source_names = [source.name for source in sources]
+        self.__data_fetched = False
+
+    def get_data(self):
+        for source in self.sources:
+            try:
+                source.get_data()
```

Comment:
Will this write to the git repo? Or will that be implemented in the child(?) class?

Reply:
This will write to the git repo yes